### ISSUE TITLE
Need way to provide the host name as an input to the CFT

### ISSUE THREAD

**Issue by C0missar** [F5Networks/f5-aws-cloudformation#99]
## Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
Describe the problem you're having or the enhancement you'd like to request.
```
The CFTs all assign a host name based on the AWS default EC2 instance name when the stack
is built.  It is not easy to change this later, as the device name is set to be the same as 
the host name, and device names cannot be changed once they have been referenced in 
a device trust without rebuilding the trust, something that is cumbersome to do via 
automation.

It would be useful to be able to specify the host name(s) during stack creation as an input 
to the CFT.
```
## Template
For bugs, enter the template with which you are experiencing issues below.

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: <Fill in level: 1 through 5>

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by shyawnkarim** [comment]
Thanks for reaching out to us with this RFE. I've created internal issue ESECLDTPLT-1974 for this.

**Comment by shyawnkarim** [comment]
Closing. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-aws-cloudformation-v2).

Parameters for inputting hostnames are available on V2.
